Kitodo/src/test/resources/ruleset_test.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
```

Comment:
Please add default Kitodo license to this file.

Reply:
done